test/test_utils/run_tests.py

```diff
@@ -1,8 +1,7 @@
 import sys
 
 if __name__ == "__main__":
-    sys.exit("This module is for import only")
-
+    raise ValueError("This module is for import only")
```

Comment:
Just a minor thing: This should probably be a `RuntimeError`. `ValueError`s are usually used to signal incorrect kinds of values that functions receive via their arguments

Reply:
Yeah, you are right, changed it with ``RuntimeError()``, thanks for review.